docs/repo/property-pages/how-to-add-a-new-property-page.md

```diff
@@ -0,0 +1,88 @@
+﻿# HOW TO: Add a New Property Page
+
+Property pages are defined by XAML files describing an instance of the [Rule](https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.framework.xamltypes.rule) class. Visual Studio uses these XAML files to dynamically build the UI and bind the controls to properties in your project. This HOW TO describes two options for defining and distributing these XAML files. At the end your property page will look something like this:
+
+![New Property Page](new-property-page.png)
+
+## Option 1: XAML file on disk
+
+In this option, the XAML resides in a standalone .xaml file on disk and is included in the end user's project as a specific kind of MSBuild item, `PropertyPageSchema`. Visual Studio reads these items from the project to determine which property pages to show.
+
+This may be an attractive option if you already have a NuGet package or Visual Studio extension that injects MSBuild .props and .targets files into the end user's project, or if you want to use MSBuild `Condition`s to control when the property page is available to a project. 
+
+### Step 1: Define the XAML file
+
+Add a new XAML file named "MyPropertyPage.xaml" to your project. Depending on how the file is created you may end up with a `<Page>` item in your project but this is not what we want as we're not using the file to describe a piece of WPF UI.
+
+Update your project to replace the `<Page>` item with one of the following:
+
+- SDK-style projects:
+  ``` xml
+  <None Update="MyPropertyPage.xaml">
+    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+  </None>
```

Comment:
For SDK-style, this style is preferred:
```suggestion
  <None Update="MyPropertyPage.xaml" CopyToOutputDirectory="PreserveNewest" />
```

Reply:
Well, it's better supported in SDK-style projects, but it's hard to call it preferred. If you update the property of a file through the Properties window (e.g. set "Copy to Output Directory" to "Copy if newer") we won't use the attribute style, even for SDK-style projects.

The only place it's really common right now is `PackageReference`.

Reply:
So, a few years back, I migrated a bunch of projects from csproj-style to SDK-style. And, at that time, I found documentation on the format with a decent migration guide. Right now, I just spent some time searching, and literally everything I found from the past is no longer there. No idea where it went, but the internet has lost information as it does over time. However, I can find remnants of why I said this is 'preferred':
- https://stackoverflow.com/a/54272143/294804
- https://github.com/dotnet/msbuild/issues/2795
- https://stackoverflow.com/a/58600845/294804

I can find answers, posts, questions, issues, etc. showing it being used as an attribute versus a node. But, I honestly cannot find the source of behind it. I would not take the actions of VS to be an 'absolute' on the format. But, I do know there had been information on what should be used as attributes versus nodes. I just can't find it anymore. 😭